app/controllers/ncr/work_orders_controller.rb

```diff
@@ -29,7 +29,7 @@ def update
 
       super
 
-      if @model_changing && !@model_instance.emergency # skip approvals if emergency
+      unless skip_state_update?
```

Comment:
how about: `if model_changing_and_not_emergency` ? It's long and blah, but it does explain what's happening!